tests/testthat/test-downloadModule.R

```diff
@@ -45,6 +45,7 @@ test_that("downloadModule downloads and unzips a parent module", {
   d_expected <- moduleMetadata("LCC2005", tmpdir)$childModules %>%
     c(m, "data", "testthat") %>% sort()
 
+#   expect_equal(length(f), 42)
   expect_equal(length(f), 44)
```

Comment:
was this change from 42 to 44 a result of you adding 2 additional files? which files are added here?